### PR TITLE
feature: TR-2486. Added new svg icons

### PR DESCRIPTION
**Pull request by svoronuk**
Related to: https://oat-sa.atlassian.net/browse/TR-2486

Added three new svg icons for navigator panel

TODO: 
- [ ] ~~Add new icons using [this](https://github.com/oat-sa/taohub-articles/blob/1d7aed57cd030fc6908b98046d05d0f159c9a1d2/forge/Tutorials/tutorial-svg-icons.md) approach~~

### CODE REVIEW

**Comment by svoronuk** [comment]
@jsconan, three svg icons were removed, because of they were not aligned between this repo and [this](https://github.com/oat-sa/tao-core-ui-fe). I don't have a lot an experience to say:
1. I remove unused icons
2.This icons are used, but in specific case, so return it back
3. Misalignment is a normal situation, so return it back
4. Misalignment is not a normal situation, so return it back and align both the repositories
What you can suggest me?

File: views/css/font/tao/tao.svg [hunk 1]
```diff
@@ -211,9 +211,6 @@
...
-<glyph unicode="&#xe915;" glyph-name="wrap-inline" d="M576 832v-448h-576v448h576zM512 768h-448v-320h448v320zM1024 448v-64h-384v64zM1024 320v-64h-1024v64zM1024 192v-64h-1024v64z" />
-<glyph unicode="&#xe916;" glyph-name="wrap-left" d="M576 704v-448h-576v448h576zM512 640h-448v-320h448v320zM1024 704v-64h-384v64zM1024 576v-64h-384v64zM1024 448v-64h-384v64zM1024 320v-64h-384v64zM1024 192v-64h-1024v64zM1024 832v-64h-1024v64z" />
-<glyph unicode="&#xe917;" glyph-name="wrap-right" d="M1024 704v-448h-576v448h576zM960 640h-448v-320h448v320zM1024 832v-64h-1024v64zM384 704v-64h-384v64zM384 576v-64h-384v64zM384 448v-64h-384v64zM384 320v-64h-384v64zM1024 192v-64h-1024v64z" />
```
Comment (olga-kulish) [inline views/css/font/tao/tao.svg:216]: wrap icons are lost

**Review by olga-kulish — COMMENTED** [review]
Question: `views/css/font/tao/tao.woff` file is updated here, but not in `tao-core-ui-fe` PR. Is it expected? I don't know how to generate font.

File: views/css/font/tao/tao.svg (hunk 1 above)
Comment (svoronuk) [reply views/css/font/tao/tao.svg:216]: I know) See comment here https://github.com/oat-sa/tao-core/pull/3214#issuecomment-958984677

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Review by olga-kulish — APPROVED** [review]
All is good.
Don't forget to update `tao-core-ui-fe` version in `views/package.json`

**Pull request merged**